modules/runner/src/main/java/org/apache/ignite/internal/configuration/storage/DistributedConfigurationStorage.java

```diff
@@ -160,11 +163,11 @@ public void close() {
                     // Meta Storage should not return nulls as values
                     assert value != null;
 
-                    if (key.equals(MASTER_KEY)) {
+                    if (Arrays.equals(key, MASTER_KEY.bytes())) {
                         continue;
                     }
 
-                    String dataKey = key.toString().substring(DISTRIBUTED_PREFIX.length());
+                    String dataKey = new String(key, StandardCharsets.UTF_8).substring(DISTRIBUTED_PREFIX.length());
```

Comment:
`UTF_8` can be imported statically decreasing the amount of syntactic noise: it is not that important that it goes from `StandardCharsets`; the important piece of information here is that it's utf-8.

Reply:
sure